docs/application-connector/docs/assets/connectorapi.yaml

```diff
@@ -97,6 +97,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/csrResponse'
+        '400':
+          description: 'Invalid CSR.'
+            content:
```

Comment:
This indentation is wrong, it should be the on same level as description, you can see the error in Swagger Editor: https://editor.swagger.io/